4788/src/main/cpp/Robot.cpp

```diff
@@ -3,60 +3,81 @@
 using namespace frc;
 using namespace wml;
 
+using hand = frc::XboxController::JoystickHand;
+
 double currentTimeStamp;
 double lastTimeStamp;
-double dt;
+double dt; //stands for delta time 
+
+//add other variables here
+double SparkSpeed;
+double TalonSpeed;
 
-// Robot Logiccd
+// Robot Logic
 void Robot::RobotInit() {
-	// Init the controllers
-	ControlMap::InitsmartControllerGroup(robotMap.contGroup);
+	
+	dt = frc::Timer::GetFPGATimestamp() - lastTimeStamp;
+	lastTimeStamp = frc::Timer::GetFPGATimestamp();
 
-	// Create wml drivetrain
-	drivetrain = new Drivetrain(robotMap.driveSystem.drivetrainConfig, robotMap.driveSystem.gainsVelocity);
+	//init controllers 
+	xbox = new frc::XboxController(0);
 
-	
-	// Zero Encoders
-	robotMap.driveSystem.drivetrain.GetConfig().leftDrive.encoder->ZeroEncoder();
-	robotMap.driveSystem.drivetrain.GetConfig().rightDrive.encoder->ZeroEncoder();
+	//Motor examples 
+	SparkMotor[0] = new frc::Spark(0);
```

Comment:
After fixing this in the header file, you can also just leave this as `SparkMotor = new frc::Spark(0);`